tests/acceptance/core/service/factory/cluster.go

```diff
@@ -0,0 +1,62 @@
+package factory
```

Comment:
Can we include this in testcases.go where all test functions could reside unless there is a reason to have it in separate directory?